### ISSUE TITLE
Should AiB hire dedicated resources for AtomOne?

### ISSUE THREAD

**Issue by moul** [atomone-hub/genesis#25]
I suggest we create a job for someone to take care of the Atomone project at AiB. This person would help Atomone move forward and be the link between Atomone’s community and AiB.

Let’s first find a leader for this job. We can think about hiring more people later if we need to.

I’m sharing this idea here, even though it’s mainly about AiB, because I think it’s good to be very open about our plans. It might encourage other groups to do the same.

**Comment by jaekwon** [comment]
Yeah AiB should invest in an AtomOne team, and also encourage others to join to make this a decentralized development ecosystem. But we want AtomOne to work with more external developers from day one. Let's get an onion going. You know what that is?

**Comment by moul** [comment]
What’s « an onion going »?

What about focusing on a single dedicated profile for now so we have someone full time without having to sacrifice other projects. Then we can help the community to hire so AiB isn’t too central.

**Comment by FarOutAndCosmic** [comment]
Strike while the iron is hot, gathering momentum by getting the talent required to build this will be well super proactive. 

The only 'onion' analogy I know is that it's grown to completion in layers. Peal one layer off at a time until at the core...

**Comment by jaekwon** [comment]
We should hire as much as we can, even if that means it's an independent team. We don't have funding limitations. One good top-level hire for example with the goal of expanding this should be fine IMO, and we can hire developers even with the expectation that before launch half will need to become independent. As long as we commit to launching it decentralized, I see no reason why we should hesitate in hiring right now. 

If you're interested in applying, please share your CV here, or DM me on Twitter. 

Strike while the iron is hot. Everyone competent who is interested and aligned should be given a job right now. This repo will coordinate all of us, whether or not AIB hires them.

**Comment by moul** [comment]
Great, I'm really excited!

I've tweeted a link to your comment: [https://twitter.com/moul/status/1730528038675255758](https://twitter.com/moul/status/1730528038675255758)

I suggest you do the same. We can repost your tweet from my account, AiB, and gnoland account.

I'll talk to the talent team about creating an official pipeline. However, I recommend starting with DMs and GitHub before using the final pipeline. Our priority is to directly engage with the people involved rather than reaching out to engineers who may not be aware of this fork.

**Comment by wnmnsr** [comment]
> If you're interested in applying, reply here, or DM me on Twitter or generally reach out.
> 
> Strike while the iron is hot. This is a red hot iron. Everyone competent who is interested and aligned should be given a job right now. 

@jaekwon I have DM'd you on Twitter. I'm here to build the AtomOne brand.

**Comment by moul** [comment]
@jaekwon, I have two initial job description proposals:

1. multiple senior golang engineer with experience with Tendermint.
2. one senior technical project manager with experience in blockchain or open-source (community-driven) projects.

We are also open to considering other potential candidates that may not strictly match these descriptions.

Please let me know your thoughts.

**Comment by stevenoruzi** [comment]
@jaekwon 

I sent you this message on X and it seems that it's in the spam directory as you have seen it:

I'm very keen on contributing to AtomOne, but I'm curious if there's a role for someone with my background. While I have more than 10 years of experience in software development, my knowledge of the technical part of crypto is quite recent and still growing. Do you have any roles for someone like me?

**Comment by AdonisVillanueva** [comment]
> We should hire as much as we can, even if that means it's an independent team. We don't have funding limitations. We can hire developers even with the expectation that before launch half will need to become independent. As long as we commit to launching it decentralized, I see no reason why we should hesitate in hiring right now.
> 
> If you're interested in applying, reply here, or DM me on Twitter or generally reach out.
> 
> Strike while the iron is hot. This is a red hot iron. Everyone competent who is interested and aligned should be given a job right now. This repo will coordinate all of us, whether or not AIB hires them. And we should be designing these founding documents such that the chain sustains them, not AIB.
> 
> We can split our Gaia fork into package-sets and define leaders for each package set, with of course some higher authority that can intervene if necessary. Lots to figure out here but I think if we figure out a good way to parallelize here without sacrificing quality, then we can make it all work very quickly. Cosmoshub4 already exists. We don't need to reinvent the wheel. If we really wanted to we could launch a minimal fork in a week.

What experience are you looking for?

**Comment by cryptulien** [comment]
@jaekwon just sent you a DM on X to help.

**Comment by moul** [comment]
"I'm brainstorming different hiring strategies, though I'm not entirely certain yet. Here are my thoughts:

1. **Hiring Leaders First**: Start by hiring leaders and managers who can then recruit and guide the rest of the team. This approach is scalable and ensures alignment in future team dynamics.

2. **Cohort Hiring for Efficiency**: Bring people on board in groups. This makes onboarding more efficient and fosters mutual support among new hires. It also allows us to discover genuine expertise, including in management, naturally shaping the organization over time.

3. **Contribution-Based Hiring**: Implement a system where contributors earn points for quality work on the repo. Negative actions (spam) could result in point deductions. Hiring decisions could be based on accumulated points, somewhat like a proof of contribution system inspired by evaluation DAO concepts on gno.land. This system is not limited for AiB, it’s like a public portfolio in the Cosmos ecosystem.

While I'm open to use standard hiring methods. Considering these unconventional methods might be beneficial for this unique project.

**Comment by tbruyelle** [comment]
@moul I like the contribution-based hiring approach. To me, it is by far the most trustworthy way to assess whether people are really committed.

**Comment by FarOutAndCosmic** [comment]
There is a definite appetite for what is being built here, it has piqued a great deal of interest and speculation. I think a centralised PR interface via 'X' for example may serve as a town square for the town criers to spread the word. Those that are interested and feel able to meaningfully contribute will quickly see the squares flag flying and see it as credible point to rally at. I would imagine some very talented help sandwiched amongst the 'wen airdrop' hunters would soon appear.

**Comment by uwezukwechibuzor** [comment]
> If you're interested in applying, reply here, or DM me on Twitter or generally reach out.
> 
> Strike while the iron is hot. This is a red hot iron. Everyone competent who is interested and aligned should be given a job right now. This repo will coordinate all of us, whether or not AIB hires them. And we should be designing these founding documents such that the chain sustains them, not AIB.

@jaekwon, I sent you a message on [X](https://twitter.com/ChibuzorWezukwe)
I'm very much interested on contributing to AtomOne

**Comment by PS92FZ** [comment]
> I'm very keen on contributing to AtomOne, but I'm curious if there's a role for someone with my background. While I have more than 10 years of experience in software development, my knowledge of the technical part of crypto is quite recent and still growing. Do you have any roles for someone like me?

Hello, all.

Commenting here on github to signal my interest as well. I second stevenoruzi's sentiment. Even if there is internship opportunity to prove qualification/and to help continue the ball rolling forward.
